plugins/command-line/prism-command-line.js

```diff
@@ -79,6 +72,22 @@
 		}
 
 		var codeLines = env.code.split('\n');
+
+		/** @type {int[]} */
+		var continuationLineIndicies = commandLine.continuationLineIndicies = new Set();
```

Comment:
The linting error here is my bad. The linter is configured to only allow ES5 code, and ES5 doesn't include `Set`. I'll add an exception for stuff that IE11 supports.

Reply:
I just updated the ESLint config. Just update this PR and the tests will pass.

Reply:
Doh! Didn't spot that in time.  Just changed the code to use a sparse array of booleans to mark the positions.  Are you happy with that rather than reverting back to a Set?

Reply:
Now back to a `Set`